src/main/java/seedu/divelog/logic/commands/ClearCommand.java

```diff
@@ -20,6 +20,9 @@ public CommandResult execute(Model model, CommandHistory history) {
         requireNonNull(model);
         model.resetData(new DiveLog());
         model.commitDiveLog();
+        if (model.getPlanningMode()) {
```

Comment:
The PR #64 adds an application state. Maybe in the future we can migrate this stuff there?